Logger/FileLogger.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Logger
+{
+    class FileLogger : BaseLogger
+    {   
+        public override void Log(LogLevel logLevel, string message)
+        {
+            Console.WriteLine(DateTime.Now + " " + ClassName + " " + logLevel + " " + message);
```

Comment:
String interpolation can simplify this
```suggestion
            Console.WriteLine($"{DateTime.Now} {ClassName} {logLevel} {message}");
```